pkg/crd/markers/crd.go

```diff
@@ -290,4 +293,23 @@ func (s Resource) ApplyToCRD(crd *apiext.CustomResourceDefinitionSpec, version s
 	return nil
 }
 
+// +controllertools:marker:generateHelp:category=CRD
+
+// UnserveVersion does not serve this version.
```

Comment:
nit: this seems a lot better as "unservedversion" (with a d).  Otherwise LGTM

Reply:
Sure, I can change this.